bot/extensions/fun/fun_cog.py

```diff
@@ -39,6 +41,20 @@ async def quote_command(self, ctx):
 
         await ctx.send(embed=embed)
 
+    @command(name='rgb', help='''Displays the RGB color entered by the user.''')
+    async def color_command(self, ctx, R: int, G: int, B: int):
+        rgb = (R, G, B)
+        img = Image.new('RGB', (200, 200), rgb)
+
+        img.save('color.png')
+        embed = Embed(
+            color=self.bot.default_color,
+            description='Here goes your color!'
+        )
+        embed.set_image(url="attachment://filename.png")
```

Comment:
If I'm not mistaken, filename needs to be the actual file name so, in this case, `color.png` and not `filename.png` [Discord.py doc about that](https://discordpy.readthedocs.io/en/latest/faq.html#how-do-i-use-a-local-image-file-for-an-embed-image ). 

Last thing is, is this image saved locally or is it just temporarily saved? If it's saved locally on the computer, be sure it's deleted afterward.

Reply:
The image seems to show just by attaching a file (without `set_image`), so I just removed those lines!